packages/html/src/listview/README.md

```diff
@@ -0,0 +1,28 @@
+## Listview
+
+```html
+<!-- default rendering -->
+<div class="k-listview k-listview-bordered   k-d-flex">
+
+    <!-- header -->
+    <div class="k-listview-header">
+        <!-- template -->
+    </div>
+
+    <!-- content -->
+    <div class="k-listview-content">
+            <!-- item template with added "k-listview-item" class to direct children -->
```

Comment:
can you please give an example of the item template dom with the k-listview-item class

Reply:
I am not sure if this would not be confused for developers and as result they will be add such element as wrapper to item template content.